bin/cli.js

```diff
@@ -76,21 +75,21 @@ program.on('--help', function() {
 var args = process.argv;
 if (!args[2]) {
   // no entry point.
-  displayWarningMessage();
-} else if (args[2] && args[2] === 'serve') {
-  // serve but with no entry point.
-  displayWarningMessage('serve');
+  showInstructions();
+} else if (args[2] === 'serve' || args[2] === 'build') {
```

Comment:
This is still not fully correct. `build` doesn't require you to provide an entry point.

Also, you need to remove the first `if` altogether. It makes no sense.